### PR TITLE
docs: changed styling for general documentation

### PR DESCRIPTION
**Pull request by harmbrugge**
What are the main changes you did:
- Changed the styling for the documentation

how to test:
- Check out the branch
- http://localhost:8080/apps/docs/#/
- See the difference

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Comment by davidruvolo51** [comment]
On the landing page, we may want to consider putting a description of each guide.  Ex:

- Platform Guide: a description of this guide
- Catalogue: ...
- Armadillo: ....

File: docs/index.html [hunk 1]
```diff
@@ -6,38 +6,91 @@
...
 <script>
     window.$docsify = {
+        name: '<img src="./img/molgenis_logo_white.png" height="50" style="margin-left: -15px;"/>',
         repo: 'http://github.com/molgenis/molgenis-emx2',
         loadNavbar: true,
         loadSidebar: true,
         relativePath: true,
         themeColor: "#005EC4",
         loadFooter: true,
-        plugins: [
```
Comment (mswertz) [inline docs/index.html:38]: why did you kill this?

**Review by mswertz — APPROVED** [review]
lovely, but why did you remove the 'edit on github'?

I mean would be great to have it at buttom. Could also part of a template?

**Comment by mswertz** [comment]
> On the landing page, we may want to consider putting a description of each guide. Ex:
> 
> * Platform Guide: a description of this guide
> * Catalogue: ...
> * Armadillo: ....

Yes please, would be great next change.

File: docs/index.html (hunk 1 above)
Comment (harmbrugge) [reply docs/index.html:38]: Revived it at the bottom of the page

**Pull request merged**